### ISSUE TITLE
Accessibility:List: The ‘Show More’ button cannot be activated using keyboard.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3484]
Description: The ‘Show More’ button cannot be activated using keyboard.

Expected: ‘Show More’ button should be activated using keyboard.

Screenshot: 

![image](https://user-images.githubusercontent.com/32538291/94648914-35fc8100-0311-11eb-8cf9-f751a2284f2c.png)
